core/audits/audit.js

```diff
@@ -215,12 +228,12 @@ class Audit {
   /**
    * @param {LH.Audit.Details.Opportunity['headings']} headings
    * @param {LH.Audit.Details.Opportunity['items']} items
-   * @param {number} overallSavingsMs
```

Comment:
We're mandating the time savings for opportunities. I've maintained previous expectation now, but may be this is something to consider. We could relax this and default to `0` when savings aren't supplied.

Reply:
`overallSavingsMs` should be required because opportunities have to be able to quantify their savings to be opportunities. We should maybe leave a comment on it to that effect so it's clearer.

(`uses-rel-preload` is the only exception because it always marks itself as not applicable to remove itself from the report while it's buggy and needs to be fixed)